amaozn clone/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Amazon Clone</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <nav>
+        <a href="index.html"> <img src="./assets/amazon_logo.png" width="100px" alt=""></a>
+        <div class="nav-country">
+            <img src="/Ecommers-for-Beginners/amaozn clone/assets/location_icon.png" height="20" alt="">
+            <div>
+                <p>Deliver to</p>
+                <h1>India</h1>
+            </div>
+        </div>
+            <div class="nav-search">
+                <div class="nav-search-category">
+                    <p>All</p>
+                    <img src="./assets/dropdown_icon.png" height="12" alt="">
+                </div>
+                <input type="text" class="nav-search-input" placeholder="search Amazon">
+                <img src="./assets/search_icon.png" class="nav-search-icon" alt="">
+            </div>
+            <div class="nav-language">
+                <img src="./assets/india flag.png" width="25px" alt="">
+                <p>EN</p>
+                <img src="./assets/dropdown_icon.png" width="8px" alt="">
+            </div>
+            <div class="nav-text">
+                <p>Hello, Sign in</p>
+                <h1>Account & Lists <img src="./assets/dropdown_icon.png" width="8px"></h1>
+            </div>
+            <div class="nav-text">
+                <p>Return</p>
+                <h1>& <img src="./assets/dropdown_icon.png" width="8px"></h1>
+            </div>
+        </div>
+        <a href="" class="nav-cart">
+            <img src="./assets/cart_icon.png" width="35px" alt="">
+            <h4>Cart</h4>
+        </a>
+
+    </nav>
+    <div class="nav-bottom">
+       <div> <img src="./assets/menu_icon.png" width="25px" alt="">
+        <p>All</p>
+    </div>
+    <p>Today's Deals</p>
+    <p>Customer service</p>
+    <p>Ragistry</p>
+    <p>Gift Card</p>
+    <p>Sell</p>
+    </div>
```

Comment:
do we need all of this?

if yes, then instead of using the para can u convert into link. can we can add empty page later and any customisation based on that.